### PR TITLE
Fix readme.md CDN notes

### PR DESCRIPTION
**Pull request by dr-vortex**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ Getting started? Play directly with the Babylon.js API using our [playground](ht
 - <https://cdn.babylonjs.com/babylon.js>
 - <https://cdn.babylonjs.com/babylon.max.js>
 
-Additional references can be found on <https://cdn.babylonjs.com/xxx> where `xxx` is the folder structure you can find in the /dist folder like <https://cdn.babylonjs.com/gui/babylon.gui.min.js>
+Additional references can be found on <https://cdn.babylonjs.com/xxx> where `xxx` is the folder structure you can find [here](https://github.com/BabylonJS/Babylon.js/tree/preview/dist) like <https://cdn.babylonjs.com/gui/babylon.gui.min.js>
```
Comment (deltakosh) [inline readme.md:20]: cc @RaananW as I guess this is not accurate anymore

Comment (RaananW) [reply readme.md:20]: The list of available packages can be found here
https://doc.babylonjs.com/divingDeeper/developWithBjs/frameworkVers#cdn-current-versions

**Review by RaananW — COMMENTED** [review]
As mentioned in the comment, the list of available packages is here https://doc.babylonjs.com/divingDeeper/developWithBjs/frameworkVers#cdn-current-versions
This is the link that should appear instead

File: readme.md (hunk 1 above)
Comment (dr-vortex) [reply readme.md]: Updated the PR.

**Review by deltakosh — APPROVED** [review]

**Pull request merged**